MovieHub/src/styles/NavigationBar.css

```
@@ -708,4 +708,145 @@
+.ai-input {
+  width: 100%;
+  padding: 0.75rem 1rem;
+  border-radius: 0.5rem;
+  border: 1px solid rgba(255,255,255,0.06);
+  background: rgba(255,255,255,0.02);
+  color: #fff;
+  outline: none;
+}
+.ai-sidebar, .ai-content { box-sizing: border-box; }
+.ai-messages { overflow-x: hidden; }
+.ai-message { max-width: 100%; word-break: break-word; white-space: pre-wrap; overflow-wrap: anywhere; }
```

Stylelint (17.2.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)